app/controllers/documents_controller.rb

```diff
@@ -9,6 +9,7 @@ class DocumentsController < AuthenticationController
   before_action :ensure_document_edits_unlocked, only: %i[new edit update archive]
 
   def index
+    @invalid_documents = @planning_application.documents.where(validated: false).where(archived_at: nil)
```

Comment:
I'd prefer to use named scopes for these where filters, in case we want to change how they work in the future. We already have a named scope for `active` so we can use that, and we can add a new one for `invalid`?

Reply:
ahhh named scopes is what @Rhian was referring to in her comment - I didn't know about these. Ok will change them.